packages/strapi-design-system/src/Pagination/components.tsx

```diff
@@ -104,45 +108,28 @@ export const NextLink = ({ children, to, ...props }) => {
 
 NextLink.displayName = 'NextLink';
 
-export const PageLink = ({ number, children, ...props }) => {
+export const PageLink = ({ number, children, ...props }: PaginationPageLinkProps) => {
   const { activePage } = usePagination();
 
   const isActive = activePage === number;
 
   return (
     <PageLinkWrapper {...props} active={isActive}>
       <VisuallyHidden>{children}</VisuallyHidden>
-      <PaginationText aria-hidden variant="pi" fontWeight={isActive ? 'bold' : null}>
+      <Typography aria-hidden fontWeight={isActive ? 'bold' : undefined} lineHeight="revert" variant="pi">
         {number}
-      </PaginationText>
+      </Typography>
     </PageLinkWrapper>
   );
 };
 
 PageLink.displayName = 'PageLink';
 
-export const Dots = ({ children, ...props }) => (
-  <DotsWrapper {...props} as="div">
+export const Dots = ({ children, ...props }: DotsProps) => (
+  <Box {...props}>
```

Comment:
This may seem a bit odd, but: previously we were using a `LinkWrapper` that was rendered as a `div`, which is contradictory. I don't see why the dots-abbreviation should be usable as link and therefore I've converted it to use a Box.

Reply:
sgtm 👍🏼